reference.txt

```diff
@@ -80,4 +80,10 @@ SECTION: IO Operations
   0x53 puts (s:str) -- Prints a string from the stack.
 
 TODO: Variable Operations
-TODO: Control Flow
+SECTION: Control Flow
+  Opcodes for controlling the program counter.
+= opcd name (params)
+==============================================================================================================
+  0x70 jump (o:int)         -- Changes the program counter by the given offset
+  0x71 cjit (o:int, c:bool) -- Changes the program counter by the given offset if the condition is met
```

Comment:
This naming choice is a bit odd to me. Why `cjit` specifically, instead of something that more obviously refers to control flow?

Reply:
Maybe jmpc for jump conditionally?

Reply:
That would probably work.